lib/src/utils/product.dart

```diff
@@ -61,6 +62,7 @@ class Subscripton {
     customerId = json['customer_id'] as String;
```

Comment:
Should `customerId` be added as an instance variable too?

Reply:
Okay, this is invalid

Reply:
`customerId` instance variable of Subscription class